src/backend/el.d

```diff
@@ -0,0 +1,221 @@
+/**
+ * Compiler implementation of the
+ * $(LINK2 http://www.dlang.org, D programming language).
+ *
+ * Copyright:   Copyright (C) 1985-1995 by Symantec
+ *              Copyright (c) 1999-2016 by Digital Mars, All Rights Reserved
+ * Authors:     $(LINK2 http://www.digitalmars.com, Walter Bright)
+ * License:     backendlicense.txt
+ * Source:      $(DMDSRC backend/_el.d)
+ */
+
+module ddmd.backend.el;
+
+import ddmd.backend.cdef;
+import ddmd.backend.cc;
+import ddmd.backend.type;
+
+import ddmd.backend.cc : Symbol;
```

Comment:
Yes it can, because two files may need to import a 3rd with the real definition, and the 3rd hasn't been pulled yet. So the first two have conflicting definitions.